<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="66788" started-at="2013-09-26T15:20:59Z" finished-at="2013-09-26T15:22:06Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="66788" started-at="2013-09-26T15:20:59Z" finished-at="2013-09-26T15:22:06Z">
      <class name="ru.st.selenium.elements.Homework3">
        <test-method status="PASS" signature="setUp()[pri:0, instance:ru.st.selenium.elements.Homework3@2c9f3eba]" name="setUp" is-config="true" duration-ms="4331" started-at="2013-09-26T16:20:59Z" finished-at="2013-09-26T16:21:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="hw3()[pri:0, instance:ru.st.selenium.elements.Homework3@2c9f3eba]" name="hw3" duration-ms="62425" started-at="2013-09-26T16:21:04Z" finished-at="2013-09-26T16:22:06Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "love"
     but: was "titanic"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "love"
     but: was "titanic"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert.assertThat(Assert.java:832)
	at ru.st.selenium.elements.Homework3.hw3(Homework3.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- hw3 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:ru.st.selenium.elements.Homework3@2c9f3eba]" name="tearDown" is-config="true" duration-ms="14" started-at="2013-09-26T16:22:06Z" finished-at="2013-09-26T16:22:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- ru.st.selenium.elements.Homework3 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
